src/propagators.jl

```diff
@@ -83,6 +83,13 @@ end
 
 QuKrylov() = QuKrylov(@compat Dict(:NC=>10))
 
+immutable QuODESolvers <: QuPropagatorMethod
+    method
```

Comment:
I am not sure it is a good idea to store the solver function like that. Maybe we can have a look how other packages realize this, but most likely they use different types. We could also use code generation to generate different types and `propagate` methods programmatically without repeating too much code.